massgov/mayflower
# React/dp 10799  create footnote and footnotelink components

## Description
This PR adds two new atom components to Mayflower-React: FootNote and FootNoteLink. 

## Related Issue / Ticket

- [DP-10799](https://jira.mass.gov/browse/DP-10799)

## Steps to Test
1.  `cd react`
2. npm start
3. go to atoms/links/FootNote and confirm expected behavior.
4.  go to atoms/links/FootNoteLink and confirm expected behavior.
